lib/tasks/port_legacy_products.rake

```diff
@@ -0,0 +1,5 @@
+task :port_legacy_products => :environment do
+  Product.where(category_id: nil).each do |product|
+    product.update_columns(category_id: Category.first.id)
```

Comment:
why are we running multiple queries for doing same work?
we can do it in one query

Reply:
done